zookeeper-docs/src/main/resources/markdown/zookeeperProgrammers.md

```diff
@@ -1513,7 +1513,7 @@ information for ZooKeeper developers.
     The definitive discussion of ZooKeeper design and performance,
     by Yahoo! Research
 
-* *API Reference _[tbd: find url]_* :
+* *[API Reference](https://zookeeper.apache.org/doc/r3.4.13/api/index.html)* :
     The complete reference to the ZooKeeper API
```

Comment:
Maybe mention explicitly to 3.4 API?

Reply:
Is "https://zookeeper.apache.org/doc/current/api/index.html" preferred?

Reply:
Changed to above